packages/static-hosting/lib/static-hosting.ts

```diff
@@ -255,6 +257,7 @@ export class StaticHosting extends Construct {
         if (props.enableErrorConfig) {
             distributionProps = {
                 ...distributionProps, ...{
+                    httpVersion: 'http3' as HttpVersion.HTTP2,
```

Comment:
I love this work around haha But can you add a TODO here to fix this once we upgrade to cdk 2?